app/services/test_application_creation_service.rb

```diff
@@ -0,0 +1,40 @@
+class TestApplicationCreationService
+  APPLICATION_TEST_TRAITS = %i[
+    test_application_initiated
+    test_application_checking_answers
+    test_application_checking_passported_answers
+    test_application_answers_checked
+    test_application_provider_submitted
+    test_application_means_completed
+    test_application_chekcing_merits_answers
+  ].freeze
+
+  PROVIDERS = Provider.all
+
+  def self.call
+    new.call
+  end
+
+  def initialize
+    @providers = Provider.all
```

Comment:
This should not be in the initializer. Create a `providers` method and use that:
```ruby
def providers
  Provider.all
end

def call
  providers.each do |provider|
```